Makefile

```diff
@@ -116,3 +116,8 @@ dev-compose-down-linux:
 
 profile:
 	pipenv run python profile_application.py
+
+generate-integration-test:
+	docker-compose -f docker-compose-dev-mac.yml up -d
+	pipenv run python -m scripts.generate_integration_test
+	pipenv run black scripts/test_*
```

Comment:
I get `Error: Invalid value for 'SRC ...': Path 'scripts/test_*' does not exist.` when trying to run this

Reply:
Fixed now.